#! /usr/bin/env python
# Task Long configuration

PACKAGE='sim_tasks'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description
#       Default    Min    Max
gen.add("angular_velocity",   double_t, 0,    "Max allowed velocity",  0.3, -1.0, 1.0)
gen.add("angle_threshold",double_t, 0,    "Angle at which the target is considered reached",  0.1)


exit(gen.generate(PACKAGE, "sim_tasks", "TaskCalibrateCompass"))

